doc/base/classes.xml

```diff
@@ -7211,7 +7211,7 @@
 			<argument index="0" name="v" type="Vector3">
 			</argument>
 			<description>
-				Return a vector transformed (multiplied) by the transposed matrix and return it. Note that this is a multiplication by inverse only when the matrix represents a rotation-reflection.
+				Return a vector transformed (multiplied) by the transposed matrix. Note that this is only a multiplication by the inverse of the matrix if it represents a rotation-reflection.
```

Comment:
I think it should be `only when` rather than `only .... if`, as it used to be, as it is clearer (IMHO).